MvvmCross-Forms/MvvmCross.Forms/Views/IMvxElement.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using MvvmCross.Binding.BindingContext;
+using MvvmCross.Core.ViewModels;
+using MvvmCross.Core.Views;
+
+namespace MvvmCross.Forms.Views
+{
+    public interface IMvxElement : IMvxView, IMvxBindingContextOwner
```

Comment:
IMvxFormsView -> IMvxElement is a breaking change, isn't it?

Reply:
Yes, the last name was wrong. Since this is just out in 5.3.1 this should not have so much impact.